openmdao/core/system.py

```diff
@@ -1246,6 +1249,57 @@ def list_states(self, stream=sys.stdout):
         else:
             stream.write("\nNo states in %s.\n" % pathname)
 
+    def list_unit_conv(self, stream=sys.stdout):
+        """ List all unit conversions that are being handled by OpenMDAO
+        (including those with units defined only on one side of the
+        connection.)
+
+        Args
+        ----
+        stream : output stream, optional
+            Stream to write the state info to. Default is sys.stdout.
+
+        Returns
+        -------
+            List of unit conversions.
+        """
+
+        params_dict = self._params_dict
+        unknowns_dict = self._unknowns_dict
+        connections = self.connections
```

Comment:
the connections dict contains ALL connections in the whole model.  All groups are just accessing the same dict, so when you loop over the connections, you need to only include those connections in our output that have at least one end contained in this group.